test/e2e/form-validation.test.js

```diff
@@ -2,13 +2,97 @@ const { assert, driver } = require('vl-ui-core').Test.Setup;
 const VlFormValidationPage = require('./pages/vl-form-validation.page');
```

Comment:
Ik ben persoonlijk niet zo een fan van help functies in test code, ik lees liever een code block op zich. Leesbaarheid primeert voor mij op duplicatie in test `it` code blokken. Voorlopig behouden we de code as is.